contribute.md

```diff
@@ -210,39 +234,42 @@ Create database tables and seed the initial data. New administrator password, in
 * SEED_ORGANIZATION
 * SEED_LOCATION
 
+Choose an environment from `config/environments` to set RAIL_ENV.
+
 ```
-bundle exec rake db:migrate
-SEED_ADMIN_PASSWORD=changeme bundle exec rake db:seed
+RAILS_ENV=test bundle exec rake db:create
+RAILS_ENV=test bundle exec rake db:migrate
+RAILS_ENV=test bundle exec rake db:seed SEED_ADMIN_PASSWORD=changeme
```

Comment:
Normally you use the `development` environment (which is default). The `test` environment is really only intended to run tests. Running tests can randomly wipe the DB which is a very frustrating development experience.